resources/test_remote.R

```diff
@@ -93,15 +93,15 @@ session = GAMBLR::get_ssh_session() # If your local machine username doesn't
 # match your GSC username, you need to add "<your_username>@gphost01.bcgsc.ca"
 # as the only argument to this function.
 test_ssm = get_ssm_by_samples(these_sample_ids = c("14-24534_tumorA","14-24534_tumorB"),
-                              ssh_session = session,subset_from_merge = T)
```

Comment:
Here, in this test set of functions the ssh session was removed throughout - can you please confirm this still works both on GSC and in remote mode?

Reply:
See pull request description at the top of this page.

Reply:
Thanks!
It was not clear if both the GSC and remote were tested but now I think this `I also ran resources/test_remote.R on GSC server` means that both setups were run. Thanks for confirming!